earth_enterprise/rpms/build.gradle

```diff
@@ -113,7 +144,7 @@ def packageInstallLibDir = new File(packageInstallRootDir, 'opt/google/lib')
 
 def postGisInstallDir = new File(
         project.projectDir,
-        '../src/NATIVE-DBG-x86_64/third_party/postgis/install'
+          "${src_dir}${native_dir()}third_party/postgis/install"
```

Comment:
Code format: This line seems to have an unequal indentation to the previous one.